scripts/upgrade

```diff
@@ -277,6 +277,17 @@ then
     chmod 640 /etc/ssl/private/dh2048.pem
 fi
 
+
+#=================================================
+# MIGRATION 5 : CREATE A .well-known FILE
+#=================================================
+
+# Create .well-known redirection for access by federation if it doesn't exist
+if yunohost --output-as plain domain list | grep -q "^$server_name$"
+then
+cp ../conf/server_name.conf /etc/nginx/conf.d/${server_name}.d/server_name.conf
+fi
```

Comment:
Same here.

- Can you put this part of code in line 214.
- Indentation 
- You also need to add the checksum management of this file here (as same as for other config files).
- You also need to replace all variables.

Reply:
I hope I did it good. I'm not so sure about replacing __SERVER_NAME__ ...